dask_sql/context.py

```diff
@@ -848,7 +848,7 @@ def _get_ral(self, sql):
             self.schema_name, case_sensitive
         )
         for schema in schemas:
-            generator_builder.addSchema(schema)
+            generator_builder = generator_builder.addSchema(schema)
```

Comment:
This was the only non-java change I made. Since this Java pattern uses a builder previously if multiple schemas were present only the last schema instance would have been saved.